iati/core/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .codelists import Code, Codelist
 from .data import Dataset
 from .rulesets import Rule, Ruleset
-from .schemas import Schema
+from .schemas import ActivitySchema, OrganisationSchema, Schema
```

Comment:
Since it shouldn't be called directly, it may make sense for `Schema` to be unavailable in shorthand form. The same would be the case for `Rule`.